src/imap/scan_folders.rs

```diff
@@ -23,7 +23,6 @@ impl Imap {
                 return Ok(());
             }
         }
-        info!(context, "Starting full folder scan");
```

Comment:
This one is probably useful?

Reply:
Idk, but it's just a single line (not one per folder), so we can leave it in